File: Programming Basics/Programming Basics - JavaScript/Exercises/03. Simple Conditions/15. 3 Equal Numbers/3EqualNumbers.js
function checkTheThreeNumbers([arg1, arg2, arg3]) {
    let num1 = Number(arg1);
    let num2 = Number(arg2);
    let num3 = Number(arg3);

    if (num1 == num2 && num2 == num3) {
        console.log("yes");
    }

    else {
        console.log("no");
    }
}

checkTheThreeNumbers(["3", "3", "3"]);
checkTheThreeNumbers(["3", "3", "1"]);
checkTheThreeNumbers(["1", "2", "3"]);

File: Programming Basics/Programming Basics - C#/Exercises/03. Simple Conditions/03. Simple Conditions/3 Equal Numbers/3 Equal Numbers.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _3_Equal_Numbers
{
    class Program
    {
        static void Main(string[] args)
        {
            var num1 = double.Parse(Console.ReadLine());
            var num2 = double.Parse(Console.ReadLine());
            var num3 = double.Parse(Console.ReadLine());

            if (num1 == num2 && num1 == num3)
            {
                Console.WriteLine("yes");
            }

            else
            {
                Console.WriteLine("no");
            }
        }
    }
}
